### PR TITLE
Initial role PR

### PR DESCRIPTION
**Pull request by haslersn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+## Example Playbook
+
+The following is an example playbook which installs and configures Riot.
+
+```yml
+- hosts: matrix01
+  become: true
+  roles:
+    - role: riot-web
```
Comment (SlothOfAnarchy) [inline README.md:17]:
```suggestion
    - role: riot-web
  vars:
```

File: defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+riot_web_write_config_to_path: /opt/riot-web/config.json
```
Comment (SlothOfAnarchy) [inline defaults/main.yml]: missing newline

File: README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+If you like to specify it as JSON, that's not an issue because any JSON is valid YAML.
+
+## Role Variable Defaults
+
+| Name                            | Default                       | Description                                                                  |
+| :------------------------------ | :---------------------------- | :--------------------------------------------------------------------------- |
+| `riot_web_archive_url`          | *required*                    | [#source-specification](#source-specification)                               |
+| `riot_web_checksum`             | *required*                    | [#source-specification](#source-specification)                               |
+| `riot_web_config`               | *required*                    | [#client-configuration](#client-configuration)                               |
+| `riot_web_write_config_to_path` | `"/opt/riot-web/config.json"` | The content of `riot_web_config` (converted to JSON) is written to this path |
```
Comment (SlothOfAnarchy) [inline README.md]: Is this quoted twice intentionally?

File: meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  company: stuvus (https://stuvus.uni-stuttgart.de)
+
+  license: AGPLv3
+
+  min_ansible_version: 2.4
+
+  platforms:
+    - name: Debian
+      versions:
+        - stretch
```
Comment (SlothOfAnarchy) [inline meta/main.yml:14]:
```suggestion
        - stretch
        - buster
```

File: tasks/install.yml [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+- name: Download Riot archive
+  get_url:
+    url: "{{ riot_web_archive_url }}"
+    dest: "{{ global_cache_dir }}/riot-web-{{ riot_web_checksum.split(':')[-1][:40] }}.tar.gz"
```
Comment (SlothOfAnarchy) [inline tasks/install.yml:5]: Please mention `global_cache_dir` in README

**Review by SlothOfAnarchy — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (haslersn) [reply README.md]: Yes

Comment (SlothOfAnarchy) [reply README.md]: Why? It doesn't need to be quoted in YAML

Comment (haslersn) [reply README.md]: The quotation marks state that it is a string (contrary to `*required*` which is meta-information).

File: README.md (hunk 1 above)
Comment (haslersn) [reply README.md:17]: This is not needed.

**Review by SlothOfAnarchy — APPROVED** [review]

**Pull request merged**